regexes/oss.yml

```diff
@@ -295,29 +295,22 @@
 ##########
 # webOS
 ##########
-- regex: 'WEBOS(\d+[\.\d]+)'
-  name: 'webOS'
-  version: '$1'
-
-- regex: 'Web0S; Linux/SmartTV.+Chrome/68'
-  name: 'webOS'
-  version: '5'
-- regex: 'Web0S; Linux/SmartTV.+Chrome/53'
-  name: 'webOS'
-  version: '4'
-- regex: 'Web0S; Linux/SmartTV.+Chrome/38'
-  name: 'webOS'
-  version: '3'
-- regex: 'Web0S; Linux/SmartTV.+Safari/538'
-  name: 'webOS'
-  version: '2'
-- regex: 'Web0S; Linux/SmartTV.+Safari/537'
-  name: 'webOS'
-  version: '1'
-
 - regex: '(?:Web0S; .*WEBOS|webOS|web0S|Palm webOS|hpwOS)(?:[/]?(\d+[\.\d]+))?'
   name: 'webOS'
   version: '$1'
+  regexs:
```

Comment:
I would suggest to rename this one into `versions` instead of `regexs` and maybe either have `version:` or `versions:`